unstract/connectors/src/unstract/connectors/filesystems/google_drive/google_drive.py

```diff
@@ -25,13 +27,16 @@ class GoogleDriveFS(UnstractFileSystem):
     # }
     def __init__(self, settings: dict[str, Any]):
         super().__init__("GoogleDrive")
-        client_secrets = json.loads(
-            GCSHelper().get_secret("google_drive_client_secret")
-        )
+        try:
+            self.client_secrets = json.loads(
```

Comment:
NIT: @kirtimanmishrazipstack why does this need to be a part of `self`?

Reply:
Because of try-catch client_secrets is not accessible in `oauth2_credentials`